config/default.js

```diff
@@ -222,6 +222,8 @@ module.exports = {
   validClientApplications: [
     'android',
     'firefox',
+    'seamonkey',
```

Comment:
Will this allow the frontend to navigate to seamonkey and thunderbird URLs? Do we want that?